models/sipnet/R/model2netcdf.SIPNET.R

```diff
@@ -74,15 +74,11 @@ model2netcdf.SIPNET <- function(outdir, sitelat, sitelon, start_date, end_date,
   num_years <- length(unique(sipnet_output$year))
   simulation_years <- unique(sipnet_output$year)
   
-  # quick consistency check
+  # get all years that we want data from
   year_seq <- seq(lubridate::year(start_date), lubridate::year(end_date))
-  if (length(year_seq) != num_years) {
-    PEcAn.logger::logger.severe("Date range specified in model2netcdf.SIPNET() function call and 
-                                total number of years in SIPNET output are not equal")
-  }
 
   # check that specified years and output years match
-  if (!all(simulation_years %in% year_seq)) {
+  if (!all(year_seq %in% simulation_years)) {
```

Comment:
:bug: :hammer: :100:

Reply:
This wasnt a bug it was a check based on some older issues with met process resulting in incorrect SIPNET years being run.

Reply:
before this check (which was done awhile ago) SIPENT runs could complete even if the wrong years or dates were simulated and this was meant to throw an consistency error.  But that was based on the older met process code where we had a lot of errors from incorrect BETY entries, etc.  But given we have been cleaning these up, I think this change should maintain a similar behavior.